config/management/genesis/src/validator_config.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_global_constants::OWNER_ACCOUNT;
+use libra_management::storage::StorageWrapper;
+use libra_management::{
+    constants,
+    error::Error,
+    secure_backend::{SharedBackend, ValidatorBackend},
+    validator_config::{create_transaction, create_validator_config_script, get_owner_from_key},
+};
+use libra_network_address::NetworkAddress;
+use libra_secure_storage::Value;
+use libra_types::{chain_id::ChainId, transaction::Transaction};
+use structopt::StructOpt;
+
+// TODO(davidiw) add operator_address, since that will eventually be the identity producing this.
+#[derive(Debug, StructOpt)]
+pub struct ValidatorConfig {
```

Comment:
I totally agree.  That would be great to merge those things together (I'm still learning all the structopt)

Another issue (at least just for me) is the overloading of names like `ValidatorConfig`.  Since this command is called `ValidatorConfig` and the object it eventually writes is the same.